clients/client-java/src/main/java/com/datastrato/gravitino/client/MessagingCatalog.java

```diff
@@ -100,7 +102,7 @@ public Topic loadTopic(NameIdentifier ident) throws NoSuchTopicException {
             ErrorHandlers.topicErrorHandler());
     resp.validate();
 
-    return resp.getTopic();
+    return fromDTO(resp.getTopic());
```

Comment:
I think it is OK to return DTO for users to use, I checked other codes like fileset, schema, it also returns DTO for users to use. Because users only know the interface, so it is OK to return a DTO to users.

Reply:
OK